lib/kennel/syncer.rb

```diff
@@ -16,7 +15,6 @@ def initialize(api, expected, project_filter: nil, tracking_id_filter: nil)
       @project_filter = project_filter
       @tracking_id_filter = tracking_id_filter
       @expected = Set.new expected # need set to speed up deletion
-      @max_diff_lines = [Integer(ENV.fetch("MAX_DIFF_LINES", "50")), 10].max
```

Comment:
The 10 was there for a reason ... but feel free to reject the reasons, they are a bit silly.

- having a negative limit makes no sense, so that shouldn't be allowed
- having a zero limit makes sense, but then breaks the "Use N * 2 to see more" message
- having a limit of 1 makes sense and workse with "N * 2", but then causes the message to mention "1 lines", which is bad grammar

A minimum of 2 sidesteps all of those issues. Raising it to 10 was an arbitrary choice, primarily to distract attention from 2 being an apparently strange choice :-p

Reply:
kk, let's do 2 then :)